src/formats/internal/reader.c

```diff
@@ -15,10 +17,16 @@ struct sd_reader {
 };
 
 sd_reader *sd_reader_open(const char *file) {
+#if !defined(_WIN32) && !defined(WIN32)
+    char path_buf[256];
+    strncpy_or_abort(path_buf, file, sizeof(path_buf));
+    if(!scan_directory_for_file(path_buf, sizeof(path_buf))) {
```

Comment:
Do we want all the sd_readers to be case insensitive, or should we only call this from the CHR loader?

Reply:
I guess all loaders case insensitive is the correct default, but is there an example where you don't want this?

Reply:
Maybe if there is some system where the operation is heavy? It could be added, yes.

Reply:
I guess we can start like this and see if anything ever breaks. My main concern is save games.

Reply:
Well, save games are a problem mainly since the font does not let you see lowercase. So, not quite sure what to do with those.